packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -985,10 +1004,12 @@ Future<T?> showMenu<T>({
       semanticLabel ??= MaterialLocalizations.of(context).popupMenuLabel;
   }
 
+  final List<GlobalKey> menuItemKeys = itemKeys ?? List<GlobalKey>.generate(items.length, (int index) => GlobalKey());
```

Comment:
If we create global keys here, don't we need to dispose them?

Reply:
Weird - I thought you needed to dispose GlobalKeys, but it looks like I'm wrong - please disregard 🙂